letsencrypt-auto-source/pieces/bootstrappers/deb_common.sh

```diff
@@ -18,11 +20,18 @@ BootstrapDebCommon() {
   # - Debian 6.0.10 "squeeze" (x64)
 
   $SUDO apt-get update || echo apt-get update hit problems but continuing anyway...
+  
+  # checking if certbot/letsencrypt is already packaged with OS
+  if [ "$QUIET" != 1 ]; then
+      if dpkg --get-selections | grep -v letsencrypt.sh | grep -c letsencrypt -gt 0; then
```

Comment:
My previous comment was about how to do what you were trying to do in a shell script. But what we should probably actually do here is this:

```
if apt-cache show certbot > /dev/null 2>&1 ; then
   echo "NOTE: Cerbot is packaged for your OS; you may want use those packages rather than cerbot-auto"
fi